src/buttons/buttons.js

```diff
@@ -24,6 +24,10 @@ angular.module('ui.bootstrap.buttons', [])
 
       //ui->model
       element.bind(buttonsCtrl.toggleEvent, function () {
+        if ('disabled' in attrs) {
+          return;
```

Comment:
In the Tabs component, this should be fine - the only way there would be a problem is if the `disabled` attribute is used instead of `disable`, which is deprecated use.

For the dropdown toggle, I opened #4033